evalml/tests/component_tests/test_arima_regressor.py

```diff
@@ -139,6 +140,7 @@ def test_set_forecast(ts_data):
     )
 
     clf = ARIMARegressor()
+    clf.last_X_index = X.index[-1]
```

Comment:
Why was this necessary?

Reply:
we never ran fit in this test so `last_X_index` wasn't being set!